pygmt/helpers/decorators.py

```diff
@@ -575,6 +575,17 @@ def new_module(*args, **kwargs):
                         f"Use long-form parameter '{long_alias}' instead."
                     )
                     warnings.warn(msg, category=SyntaxWarning, stacklevel=2)
+
+            # timestamp (U) is deprecated since v0.8.0, remove in v0.12.0
+            if "U" in kwargs or "timestamp" in kwargs:
+                if "timestamp" in kwargs:
+                    kwargs["U"] = kwargs.pop("timestamp")
+                msg = (
+                    "Parameter 'U' and 'timestamp' is deprecated since v0.8.0. "
+                    "Use Figure.timestamp() instead."
```

Comment:
```suggestion
                    "Parameters 'U' and 'timestamp' are deprecated since v0.8.0 "
                    "and will be removed in v0.12.0. "
                    "Use Figure.timestamp() instead."
```